src/Migrations/M034MigrateCoversContentBlockToPostsListBlock.php

```diff
@@ -0,0 +1,422 @@
+<?php
+
+// phpcs:disable Generic.Files.LineLength.MaxExceeded
+
+namespace P4\MasterTheme\Migrations;
+
+use P4\MasterTheme\MigrationRecord;
+use P4\MasterTheme\MigrationScript;
+use WP_Block_Parser;
+
+/**
+ * Migrate Covers block of type Content to Posts List blocks.
+ */
+class M034MigrateCoversContentBlockToPostsListBlock extends MigrationScript
+{
+    /**
+     * Perform the actual migration.
+     *
+     * @param MigrationRecord $record Information on the execution, can be used to add logs.
+     * phpcs:disable SlevomatCodingStandard.Functions.UnusedParameter -- interface implementation
+     */
+    protected static function execute(MigrationRecord $record): void
+    {
+        try {
+            // Get the list of posts using Covers blocks.
+            $posts = Utils\Functions::get_posts_using_specific_block(Utils\Constants::BLOCK_COVERS, Utils\Constants::ALL_POST_TYPES);
+
+            // If there are no posts, abort.
+            if (!$posts) {
+                return;
+            }
+
+            foreach ($posts as $post) {
+                // Get all the blocks of each post.
+                $parser = new WP_Block_Parser();
```

Comment:
We can keep the `WP_Block_Parser` initialization outside the loop, which can help not to instantiating it repeatedly in each iteration.

Reply:
Fixed @sagarsdeshmukh, thanks